doc/sphinx/source/frameworks/validated/list.rst

```diff
@@ -1,24 +1,23 @@
 .. frameworks/validated/list.rst: 
 
-#################################
-Validated workloads by framework
-#################################
+.. _validated:
 
-We validated performance [#f1]_ for the following TensorFlow\* and MXNet\* 
-workloads:
 
-* :ref:`tensorflow_valid`
-* :ref:`mxnet_valid`
-* :ref:`onnx_valid`
-* :ref:`testing_latency`
+Validated workloads
+###################
 
-.. _tensorflow_valid:
+We validated performance [#f1]_ for the following workloads:
```

Comment:
I think `We have validated ...` sounds better here.

Reply:
Done